src/helpers/isFileFormattable.js

```diff
@@ -8,13 +9,26 @@ const { findCachedFromFilePath } = require('./general');
 const getNearestPrettierignorePath = (filePath: FilePath): ?FilePath =>
   findCachedFromFilePath(filePath, '.prettierignore');
 
+const getNearestPrettierignoreParentDirPathRelativeToCurrentFilePath = (editor: TextEditor): string => {
```

Comment:
Sorry this was my fault again, I got the name like backwards, it should be: `getCurrentFilePathRelativeToNearestPrettierignoreParentDir`